components/ui/useCode/Challenge-FormField.tsx

```diff
@@ -0,0 +1,53 @@
+import {
+    FormControl,
+    FormField,
+    FormItem,
+    FormLabel,
+    FormMessage,
+} from "@/components/ui/form";
+import {
+    Select,
+    SelectContent,
+    SelectItem,
+    SelectTrigger,
+    SelectValue,
+} from "@/components/ui/select";
+
+export function ChallengeFormField(form: any) {
+    return (
+        <FormField
+            control={form.control}
+            name='challenge'
+            render={({ field }) => (
+                <FormItem className='mb-[1px]'>
+                    <FormLabel>Challenge/Level</FormLabel>
+                    <FormControl>
+                        <Select
+                            onValueChange={field.onChange}
+                            defaultValue={field.value}
+                        >
+                            <SelectTrigger className='w-[180px]'>
+                                <SelectValue placeholder='Select a Challenge' />
+                            </SelectTrigger>
+                            <SelectContent>
+                                <SelectItem value='helloWorld'>
+                                    helloWorld
+                                </SelectItem>
```

Comment:
Hard coding here. I assume this isn't what the final implementation looks like. 

An idea would be to declare an exported constant enum of challenges, then use a `.map()` function to iterate the list of challenges.`components/landing/test-challenges/challenge-code.ts` could benefit from declaring this enum too.

Reply:
done in a way, not sure if the way I did it is the optimal way, if you have any feedback on the change, feel free to lemme know